requirements.txt

```diff
@@ -3,22 +3,29 @@ anyio==4.9.0
 app-common-python==0.2.7
 certifi==2025.4.26
 click==8.2.1
+distlib==0.3.9
 dnspython==2.7.0
 email_validator==2.2.0
 fastapi==0.115.12
 fastapi-cli==0.0.7
+filelock==3.17.0
 greenlet==3.2.2
+grpcio==1.73.0rc1
 h11==0.16.0
 httpcore==1.0.9
 httptools==0.6.4
 httpx==0.28.1
 idna==3.10
+inventory-grpc-clients-python-kessel-project==0.10.14
 Jinja2==3.1.6
 markdown-it-py==3.0.0
 MarkupSafe==3.0.2
 mdurl==0.1.2
+pipenv==2024.4.1
```

Comment:
mmh. I wonder why pipenv and virtualenv are being added as dependencies of this project. :thinking:

Reply:
Thanks for fixing this.